agent.go

```diff
@@ -847,7 +847,9 @@ func mountToRootfs(m initMount) error {
 func generalMount() error {
 	for _, m := range initRootfsMounts {
 		if err := mountToRootfs(m); err != nil {
-			return err
+			// Don't fail and try to mount the rest of filesystems
+			agentLog.Warnf("Could not mount %v: %v", m.src, err)
```

Comment:
If we are sure `/dev` is the only exception, can you filter it out and still fail for all the other cases?